jetbrains-gateway/src/software/aws/toolkits/jetbrains/gateway/connection/caws/CawsCommandExecutor.kt

```diff
@@ -15,7 +15,7 @@ class CawsCommandExecutor(
     ssmTarget: String,
     private val spaceName: String,
     private val projectName: String
-) : AbstractSsmCommandExecutor(REGION, ssmTarget) {
+) : AbstractSsmCommandExecutor(AwsRegion.GLOBAL, ssmTarget) {
```

Comment:
is this the region used for `https://public.codecatalyst.global.api.aws` endpoint? what does `AwsRegion.GLOBAL` resolve to? Wondering if we need to make a change in [vscode toolkit](https://github.com/aws/aws-toolkit-vscode/blob/06507c380c67c5aec7f3cf3f66e90e0f622232ca/src/shared/clients/codecatalystClient.ts#L53-L54).

Reply:
since the endpoint is hardcoded, it appears you can set the region to whatever you want and it's ignored by the SDKs.
this line defines what region is passed to session-manager-plugin, which also appears to be ignored